Modify some kettles to fix behavior of invisible items

JoseTomasTocino opened the pull request.

JoseTomasTocino committed:
Modify some kettles to fix behavior of invisible items

bluezio commented:
Could you explain which changes you did? The diff doesn't help much, I'm afraid.

JoseTomasTocino commented:
Sure. The problem was that administrators were not being properly detected. In order to fix that two files were modified.

On the one hand, the `_list-child-sections` file. We added an additional `isAdmin4` field to the `isAdmin x2` step. In the `roots (unfiltered)` step that follows, we added an additional condition to the `WHERE` clause so that it checks if `isAdmin` is true. Same with the step `children (unfiltered)`. The rest is the same.

On the other hand, in the `_list-resources-subsections` file, the `isAdmin` step was modified to improve the regular expressions that detects Administrators.

bluezio commented:
Thanks! I'm sorry that it took some time until I could review these changes :-/. They look fine to me. I'll release 1.0.1 with the fix, then.
